tools/WinObjC.Tools/vsimporter-templates/packageable.project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "WinObjC.Packaging": "*",
```

Comment:
Yes, but project.json can be also for iOS middleware that does not wish to be packageable. Although I'd assume most middleware would like to be packaged, this gives them the option to opt out of packaging.

Currently, the default for importing any project is to not add the packaging project to the solution, ie. make it not packageable. 
However, projects imported with -genprojections will automatically add the packaging project to the solution.
Should the default instead be to include the packaging project?

---
In reply to: [98952538](https://github.com/Microsoft/WinObjC/pull/1880#discussion_r98952538) [](ancestors = 98952538)